docs/samples/pipelines/sample-custom-model.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import kfp.dsl as dsl
+from kfp import components
+import json
+
+
+kfserving_op = components.load_component_from_file('component.yaml')
```

Comment:
what's `component.yaml` ?

Reply:
defines what will serve the request in pipeline. Added a todo for updating with raw github URL from KFP repo once the component is updated there